.github/workflows/ci.yml

```diff
@@ -112,6 +112,18 @@ jobs:
           go install github.com/goreleaser/nfpm/v2/cmd/nfpm@${{ env.NFPM_VERSION }}
           OS_RELEASE="${{ matrix.container.image }}" OS_VERSION="${{ matrix.container.version }}" \
             make integration-test
+      - name: Archive install/uninstall integration test logs
+        uses: actions/upload-artifact@v3
+        with:
+          name: integration-test-log${{ matrix.container.image }}-${{ matrix.container.version }}
```

Comment:
Think the name may be missing an additional dash in it. Could you update it to this `integration-test-log-${{ matrix.container.image }}-${{ matrix.container.version }}`